### PR TITLE
WV-2857 Additional Properties GITC

### PR DESCRIPTION
**Pull request by ryanweiler92**
## Description

Updates the validateConfigs build file to set the `additionalProperties` property to true for GITC builds.

## How To Test

1. `git checkout wv-2857-additional-properties-gitc`
2. `npm ci`
3. `npm run build`
4. `npm run watch`
5. Verify that WV still built correctly.

@nasa-gibs/worldview

### CODE REVIEW

File: tasks/build-options/validateConfigs.js [hunk 1]
```diff
@@ -68,7 +74,8 @@ async function validateFile (filePath) {
     for (const error of validate.errors) {
       invalidJsonFiles.push(error)
       console.error(`${prog}: ERROR: ${error.instancePath} ${error.message}`)
-      if (argv.mode === 'verbose') console.error(error)
+      // TOD: Add verbose mode with the full error:
```
Comment (PatchesMaps) [inline tasks/build-options/validateConfigs.js:77]: Is this change related to the GITC stuff?

Comment (ryanweiler92) [reply tasks/build-options/validateConfigs.js]: I have removed these with the latest commit

**Review by Tomcariello — APPROVED** [review]

**Review by PatchesMaps — APPROVED** [review]

**Review by minniewong — APPROVED** [review]
